src/main/app/package-lock.json

```diff
@@ -0,0 +1,15684 @@
+{
```

Comment:
Hey @JustinLohKingWei, so this package-lock.json is the lock file that `npm` uses. It was generated when you added a dependency with npm. Since we are using yarn, we don't need this file so can you delete it?

Reply:
Package-lock.json has been removed